### PR TITLE
feat:add k8s in k8s websocket node agent for unjoin and join node

### PR DESCRIPTION
**Pull request by village-way**
#### What type of PR is this?

/kind feature

#### What does this PR do?

add node agent for k8s in k8s, the agent have this feature
1. support execute shell command
2. support execute python command
3. support execute shell script
4. support upload file to the node 
5. support http basic auth
6. support self signed cert
#### Which issue(s) does this PR fix?

Fixes #
#### Special notes for your reviewer:

@OrangeBao @yuleichun-striving @duanmengkk 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: web-cmd/app.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    return_code = -1
+    # create temp file and delete the file outside the with seq
+    with tempfile.NamedTemporaryFile(delete=True) as temp:
+        temp.write(data.encode('utf-8'))
+        temp.flush()  # flush data to disk
+        # get file name
+        file_path = temp.name
+        logger.info(file_path)
+        with subprocess.Popen(['bash', file_path],
+                              stdout=subprocess.PIPE,stderr=subprocess.STDOUT,
```
Comment (OrangeBao) [inline web-cmd/app.py]: Need to pass parameters to the script.The script execution case is as follows:
```bash kubelet_node_helper.sh join dns_address```
```bash kubelet_node_helper.sh logs kubelet```

Comment (village-way) [reply web-cmd/app.py]: fixed. use client.py test it work fine, you should use the url path to pass the args

**Comment by duanmengkk** [comment]
fix the lint error @village-way

**Comment by village-way** [comment]
> fix the lint error @village-way

fixed

**Pull request merged**